specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimportalsettings.json

```diff
@@ -0,0 +1,590 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delegation settings.",
+            "schema": {
+              "$ref": "#/definitions/PortalDelegationSettings"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "DelegationSettings"
+        ],
+        "operationId": "DelegationSettings_Update",
+        "description": "Update Delegation settings.",
+        "x-ms-examples": {
+          "ApiManagementPortalSettingsUpdateDelegation": {
+            "$ref": "examples/ApiManagementPortalSettingsUpdateDelegation.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PortalDelegationSettings"
+            },
+            "description": "Update Delegation settings."
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": true,
+            "description": "The entity state (Etag) version of the property to update. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+            "type": "string"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Delegation settings was updated successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "DelegationSettings"
+        ],
+        "operationId": "DelegationSettings_CreateOrUpdate",
```

Comment:
Add this file to the readme.md under 2017-03-01

Reply:
Done.